server/services/admin/slug/checkSlugAvailability.ts

```diff
@@ -0,0 +1,23 @@
+import { Event } from '../../../models/event';
+
+export interface AdminCheckSlugResponse {
+  id: Event['id'] | null;
+  title: string | null;
+}
+
+/**
+ * @param slugOrId Event id if admin === true, slug if admin === false.
+ * @param admin Whether or not to return results for the admin view.
```

Comment:
What are these related to? The function only has argument `slug`

Reply:
The Copypaste Strikes Back... fixed.